components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -525,7 +538,7 @@ void RewardsServiceImpl::MaybeShowAddFundsNotification(
   // next 3 days and balance is too low.
   base::Time now = base::Time::Now();
   if (reconcile_stamp - now.ToDoubleT() <
-      3 * base::Time::kHoursPerDay * base::Time::kSecondsPerHour) {
+      3 * base::Time::kHoursPerDay * BASE_TIME_KSECONDSPERHOUR) {
```

Comment:
why can't we use base here?

Reply:
that is again what I described about between chromium transitions.

Reply:
Fixed